app/models/developer.rb

```diff
@@ -18,9 +18,9 @@ class Developer < ApplicationRecord
   validates :name, presence: true
   validates :hero, presence: true
   validates :bio, presence: true
-  validates :avatar, content_type: ["image/png", "image/jpg", "image/jpeg"],
+  validates :avatar, content_type: ["image/png", "image/jpeg", "image/jpg"],
     max_file_size: 2.megabytes
-  validates :cover_image, content_type: ["image/png", "image/jpg", "image/jpeg", "image/gif"],
+  validates :cover_image, content_type: ["image/png", "image/jpeg", "image/jpg", "image/gif"],
```

Comment:
Do you remember why we accept a gif for the cover image and not for the avatar?

I just checked production and no one has uploaded a gif - let's remove this accepted format.

Reply:
Honestly, I think I was thinking of how Dev.to does it when I added that validation. It was totally arbitrary aside from that though. We never really discussed it.